source/math/FloatComplexVectorList.ooc

```diff
@@ -77,6 +50,33 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 			result
 		}
 	}
+	init: func ~default {
+		super()
+	}
+	init: func ~capacity (capacity: Int) {
+		super(capacity)
+	}
+	init: func ~fromVectorList (other: VectorList<FloatComplex>) {
+		super(other _vector)
+		this _count = other count
+	}
+	init: func ~withValue (capacity: Int, value: FloatComplex) {
+		super(capacity)
+		for (i in 0 .. capacity)
+			this add(value)
+	}
+	toVectorList: func -> VectorList<FloatComplex> {
+		result := VectorList<FloatComplex> new()
+		result _vector = this _vector
+		result _count = this _count
+		result
+	}
+	copy: func -> This {
+		result := This new(this _count)
+		for (i in 0 .. this _count)
+			result add(this[i])
+		result
+	}
```

Comment:
Any changes to the code, syntax-wise, should not be in this PR, as it will clutter them up and also make them huge.